src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -40,7 +40,8 @@ limit.
 For example Neuromag/Elekta/Megin, which can produce several files
 for a single recording.
 Both `some_file.fif` and `some_file-1.fif` would belong to a single recording.
-In BIDS, the [`split`](../99-appendices/09-entities.md#split) entity is RECOMMENDED to deal with split files.
+In BIDS, the [`split`](../99-appendices/09-entities.md#split) entity is RECOMMENDED to deal with split files. 
+If there are split files, they SHOULD be listed as separate recordings in the `scans.tsv` file.
```

Comment:
Why make it a SHOULD when it could also be a MUST? :) Just saying…

Reply:
I was thinking for backward compatibility reasons. But one could make the case that because the case was unclear before (whether to put each split individually, or only a single split into scans), the usual BIDS rules must have applied (putting each scan individually), and thus this is a backward compatible clarification that removes an unfortunate ambiguity.

So +1 to change SHOULD to MUST

Reply:
Yes my POV is also: it was unclear / unspecified before, now let's not make the mistake to leave us in this miserable situation by making it a wobbly SHOULD. -> Let's go for the MUST.

Reply:
MUST it is then